src/main/java/io/github/thebusybiscuit/hotbarpets/groups/PeacefulAnimals.java

```diff
@@ -48,8 +48,16 @@ public void load(HotbarPets plugin) {
 				new ItemStack(Material.SALMON), new ItemStack(Material.EMERALD), new ItemStack(Material.SALMON),
 				new ItemStack(Material.LAPIS_LAZULI), new ItemStack(Material.COD), new ItemStack(Material.LAPIS_LAZULI)
 		}).register();
-	}
-
-
 
+		new HotbarPet(plugin.getCategory(), new SlimefunItemStack(
+				"HOTBAR_PET_PANDA",
+				"414ff627a6a6f35e1d717ebcb191e4c7f9097542db599e7108ae2c7dd3513e51",
```

Comment:
Come on, change this to Base64 or change all the other ones to Hashes...
But don't be so inconsistent.